pkgs/development/python-modules/detect-secrets/default.nix

```diff
@@ -36,6 +38,7 @@ buildPythonPackage rec {
     pytestCheckHook
     responses
     unidiff
+    pkgs.gitMinimal
```

Comment:
Open question - is it better to require `pkgs` to pull in non-python dependencies or just accept a `git` arg and explicitly set it in the `python-packages.nix` call?

Reply:
I followed the manual using `pkgs`: https://nixos.org/manual/nixpkgs/unstable/#handling-dependencies